forage-android/src/main/java/com/joinforage/forage/android/core/services/forageapi/network/OkHttpClientBuilder.kt

```diff
@@ -19,7 +19,8 @@ internal object OkHttpClientBuilder {
         sessionToken: String,
         merchantId: String? = null,
         idempotencyKey: String? = null,
-        traceId: String? = null
+        traceId: String? = null,
+        apiVersion: String = "default"
```

Comment:
`provideOkHttpClient` get's used by a lot of callees. Declaring the `apiVersion` with a param that defaults to `default` will decouple these callers from needing to work directly with `OkHttpClient.addHeader` and the `ForageConstants.Headers`. Less coupling makes for more maintainable code!